test/Parse/consecutive_statements.swift

```diff
@@ -7,7 +7,7 @@ func statement_starts() {
   f(0)
   f (0)
   f // expected-error{{expression resolves to an unused l-value}}
-  (0)
+  (0) // expected-warning {{result of call to 'init(_builtinIntegerLiteral:)' is unused}}
```

Comment:
Ick. We should offer a better diagnostic for implicit calls. That can come later, though.